### ISSUE TITLE
Error message after posting a sticker

### ISSUE THREAD

**Issue by eivindm** [tim-sueberkrueb/ubuntu-hangups#42]
![screenshot_bug_hangups](https://cloud.githubusercontent.com/assets/78692/12128380/38e19e5e-b3fd-11e5-9071-aa07445a15d0.png)
Got this error message while sending a sticker.  Had to kill the and restart the app.  I was not able to reproduce the error.

**Comment by timsueberkrueb** [comment]
Looks like it lost the connection, see #7

**Comment by eivindm** [comment]
Could be.  But I was at home on a connection that is normally very stable.  So in that case I suspect this can happen for some reason even though you have a stable connection (hangout server problem or something), so it could be worth the effort to handle the error and present a good error message.  

By the way, my wife got the sticker I sent, so it was able to send, but not receive the response after posting.

**Comment by timsueberkrueb** [comment]
@eivindm okay, yes, when offline support gets integrated this should be prevented. I'll think about adding a better error message.

**Comment by bittner** [comment]
@eivindm The error message popup is scrollable, though.  Try with two fingers. It shouldn't be necessary to kill the app.

Of course, a non-blocking error status message (e.g. in a toolbar or footer) instead of a blocking popup, ideally in general for all errors or warnings, would probably be less disturbing.

**Comment by timsueberkrueb** [comment]
@bittner yes that's better, an error message like this should be only used if something irreversible goes wrong :)
